apps/astarte_appengine_api/lib/astarte_appengine_api/device/queries.ex

```diff
@@ -121,6 +121,24 @@ defmodule Astarte.AppEngine.API.Device.Queries do
     |> Mapping.from_db_result!()
   end
 
+  def retrieve_all_mappings(db_client, interface_id) do
```

Comment:
We already have `Astarte.DataAccess.Mappings.fetch_interface_mappings` function. Why aren't we using it?